libnmstate/nm/nmclient.py

```diff
@@ -51,6 +51,17 @@ def client(refresh=False):
     if _nmclient is None or refresh:
         if NM:
             _nmclient = NM.Client.new(None)
+            # if get_state() returns 0 the NM state is unknown
+            if _nmclient.get_state() == 0:
+                logging.error(
+                    'NetworkManager is in unknown state, '
```

Comment:
I guess that if it is in an unknown state, it means that it is not running.
A user will understand "not running" over "unknown" state.

Reply:
I was not sure if omitting state's details would be fine but yes, makes sense. However as now it checks directly for running status, makes even more sense. 
 In the latest [commit b04e4b5](https://github.com/nmstate/nmstate/commit/b04e4b5d0135520d5925683c7da46ba1415ebd71) I forgot to replace that. I'll provide new changes tomorrow.